reference/constraints/Callback.rst

```diff
@@ -5,7 +5,7 @@ The purpose of the Callback constraint is to create completely custom
 validation rules and to assign any validation errors to specific fields
 on your object. If you're using validation with forms, this means that you
 can make these custom errors display next to a specific field, instead of
-simply at the top of your form.
+the top of your form.
```

Comment:
**at** the top? as it indicates a position.

Reply:
You are right!

Reply:
"instead of at the top" ? But I guess a native speaker just would not say it like this because of the "of at".
"instead at" is wrong IMHO.